_config.yml

```diff
@@ -1,6 +1,8 @@
 markdown: kramdown
 paginate: 10
 
+theme: materialize
```

Comment:
this may need to be remote theme
https://blog.github.com/2017-11-29-use-any-theme-with-github-pages